worker.js

```diff
@@ -93,6 +97,26 @@ exports.initialize = function(id, env) {
 };
 
 /**
+ * Set up any data transformation if needed
+ *
+ */
+exports.initialize_transform = function() {
+    if (exports.env.options.xform && exports.env.options.xform.file) {
+        try {
+            exports.transform_function = encapsulator(exports.env.options.xform.file, 'transform').transform;
+            log.info(">> Data transform will be done during export using function file " + exports.env.options.xform.file);
```

Comment:
This is really going to spam the command line if we print this for every single hit. I'm not sure it's a good idea to do it this way. Rather than that I would have one debug line somewhere that shows wether a transform is enabled or not.